go-controller/pkg/cni/helper_linux.go

```diff
@@ -307,6 +309,29 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 	return []*current.Interface{hostIface, contIface}, nil
 }
 
+func (pr *PodRequest) deletePodConntrack() {
+	if pr.CNIConf.PrevResult == nil {
+		return
+	}
+	result, err := current.NewResultFromResult(pr.CNIConf.PrevResult)
+	if err != nil {
+		klog.Warningf("could not convert result to current version: %v", err)
+		return
+	}
+
+	for _, ip := range result.IPs {
+		// Skip known non-sandbox interfaces
```

Comment:
@squeed @dcbw can you guys review this block? I'm not sure why this skip is here but I see similar logic in:

https://github.com/containernetworking/cni/pull/378/files#diff-5d88fb75314f3a2ca2345bc9d0777e66R93

Reply:
@trozet the CNI plugin may return the IP address of host-side interfaces (like the bridge). You only want to perform these operations on the container IP address. In reality ovnkube doesn't return anything other than the container interface right now, but better to be correct from the start.